tools/fpgabist/bist/README

```diff
@@ -0,0 +1,7 @@
+The Arria® 10 DCP BIST is a mechanism designed to allow 
+the DCP system to troubleshoot itself and find components 
```

Comment:
Is this limited to only DCP systems? Would it not work for Xeon+Integrated FPGA?